src/lib/mdns/Discovery_ImplPlatform.h

```diff
@@ -54,6 +54,11 @@ class DiscoveryImplPlatform : public ServiceAdvertiser, public Resolver
     /// Requests resolution of a node ID to its address
     CHIP_ERROR ResolveNodeId(const PeerId & peerId, Inet::IPAddressType type) override;
 
+    CHIP_ERROR FindCommissionableNodes(CommissionableNodeFilter filter = CommissionableNodeFilter()) override
+    {
+        return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
Any reason not to implement this?

Reply:
Just haven't gotten there yet - it's not currently part of my test setup. Will add an issue.